.github/workflows/test.yml

```diff
@@ -29,12 +33,19 @@ jobs:
           cache-dependency-path: "wormhole-connect/package-lock.json"
       - name: Install dependencies
         run: npm ci
-      - name: Run Playwright tests
-        run: npm run test:e2e
+      - name: Run Playwright tests with wallet
+        if: ${{ startsWith(github.event.inputs.base_url, 'http://localhost') || github.event_name == 'schedule' }}
+        run: npm run test:e2e -- --grep-invert @noWallet
         env:
-          BASE_URL: ${{ github.event.inputs.base_url }}
+          BASE_URL: ${{ github.event.inputs.base_url || 'http://localhost:5173' }}
+          REACT_APP_TEST_CG_AK: ${{ secrets.REACT_APP_TEST_CG_AK }}
           REACT_APP_TEST_EVM_ADDR: ${{ secrets.REACT_APP_TEST_EVM_ADDR }}
           REACT_APP_TEST_EVM_PK: ${{ secrets.REACT_APP_TEST_EVM_PK }}
+      - name: Run Playwright tests with no wallet
+        if: ${{ !startsWith(github.event.inputs.base_url, 'http://localhost') ||  github.event_name == 'pull_request' }}
+        run: npm run test:e2e -- --grep @noWallet
```

Comment:
minor but IMO would be cleaner to just give each of these their own `script`.

`npm run test:e2e:with-wallet`

`npm run test:e2e:without-wallet`

passing cmd like flags into an npm script just kinda confusing